pkg/config/config.go

```diff
@@ -145,6 +150,15 @@ func (settings *ConfigurationSettings) postReadAdjustments() error {
 	if submatch := envVariableRegexp.FindStringSubmatch(settings.BackendMySQLPassword); len(submatch) > 1 {
 		settings.BackendMySQLPassword = os.Getenv(submatch[1])
 	}
+	if submatch := envVariableRegexp.FindStringSubmatch(settings.BackendMySQLTlsCaCertPath); len(submatch) > 1 {
+		settings.BackendMySQLTlsCaCertPath = os.Getenv(submatch[1])
+	}
+	if submatch := envVariableRegexp.FindStringSubmatch(settings.BackendMySQLTlsClientCertPath); len(submatch) > 1 {
+		settings.BackendMySQLTlsClientCertPath = os.Getenv(submatch[1])
+	}
+	if submatch := envVariableRegexp.FindStringSubmatch(settings.BackendMySQLTlsClientKeyPath); len(submatch) > 1 {
+		settings.BackendMySQLTlsClientKeyPath = os.Getenv(submatch[1])
+	}
```

Comment:
We introduced `envVariableRegexp` so as to allow users to pass credentials as environment variables rather than plaintext in a config file. I think this is not required for configuration like TLS paths, but perhaps I'm not aware that this may be sensitive data?

Reply:
Yeah, this comes from a use case at Fastly, we're a common pattern is to express sensitive file paths as environment variables that are lately populated by chef. I thought it wouldn't hurt, but I'm not particularly strong about leaving it either.

Reply:
Certainly doesn't hurt to keep this. If there's a use case, then it makes perfect sense!